server/methods/deleteMessage.coffee

```diff
@@ -18,7 +17,7 @@ Meteor.methods
 
 		deleteQuery = 
 			_id: message._id
-		deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
+		#deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
```

Comment:
No, if user is a normal user, just pass user to delete query as the commented code
`deleteQuery['u._id'] = Meteor.userId()`